apps/docs/spec/supabase_py_v2.yml

```diff
@@ -64,91 +155,267 @@ functions:
+            "session": {
+              "provider_token": null,
+              "provider_refresh_token": null,
+              "access_token": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJhdXRoZW50aWNhdGVkIiwiZXhwIjoxNzE4NTg3MTY1LCJpYXQiOjE3MTg1ODM1NjUsImlzcyI6Imh0dHA6Ly8xMjcuMC4wLjE6NTQzMjEvYXV0aC92MSIsInN1YiI6ImU0MTBkMTBmLTZlNTktNDBlNS1hMmRlLTY5NGE5MzVlNzJlZCIsImVtYWlsIjoiaGVsbG9AZXhhbXBsZS5jb20iLCJwaG9uZSI6IiIsImFwcF9tZXRhZGF0YSI6eyJwcm92aWRlciI6ImVtYWlsIiwicHJvdmlkZXJzIjpbImVtYWlsIl19LCJ1c2VyX21ldGFkYXRhIjp7fSwicm9sZSI6ImF1dGhlbnRpY2F0ZWQiLCJhYWwiOiJhYWwxIiwiYW1yIjpbeyJtZXRob2QiOiJwYXNzd29yZCIsInRpbWVzdGFtcCI6MTcxODU4MzU2NX1dLCJzZXNzaW9uX2lkIjoiMjRmOGFlMmMtYWM4Mi00NzdhLWIxMTMtZTkwNmRlNzFiMDIxIn0.5KrE1AL4koBJs5dvNQfThMV1HxV8nPP-GPU24aRL9xI",
+              "refresh_token": "mNUp4-LDuYq-kNG61t-2Xw",
+              "expires_in": 3600,
+              "expires_at": 1718587165,
+              "token_type": "bearer",
+              "user": {
+                "id": "11111111-1111-1111-1111-111111111111",
+                "app_metadata": {
+                  "provider": "email",
+                  "providers": [
+                    "email"
+                  ]
                 },
-                provider='email', 
-                created_at=datetime.datetime(2024, 1, 1, 1, 0, 0, 0, tzinfo=TzInfo(UTC)), 
-                last_sign_in_at=datetime.datetime(2024, 1, 1, 1, 0, 0, 0, tzinfo=TzInfo(UTC)), 
-                updated_at=datetime.datetime(2024, 1, 1, 1, 0, 0, 0, tzinfo=TzInfo(UTC))
-              )
-            ],
-            factors=None
-          ) 
-          session=None
+                "user_metadata": {},
+                "aud": "authenticated",
+                "confirmation_sent_at": null,
+                "recovery_sent_at": null,
+                "email_change_sent_at": null,
+                "new_email": null,
+                "invited_at": null,
+                "action_link": null,
+                "email": "hello@example.com",
+                "phone": "",
+                "created_at": "2024-06-17T00:19:25.760110Z",
+                "confirmed_at": null,
+                "email_confirmed_at": "2024-06-17T00:19:25.779181Z",
+                "phone_confirmed_at": null,
+                "last_sign_in_at": "2024-06-17T00:19:25.785489Z",
+                "role": "authenticated",
+                "updated_at": "2024-06-17T00:19:25.794650Z",
+                "identities": [
+                  {
+                    "id": "11111111-1111-1111-1111-111111111111",
+                    "user_id": "11111111-1111-1111-1111-111111111111",
+                    "identity_data": {
+                      "email": "hello@example.com",
+                      "sub": "11111111-1111-1111-1111-111111111111"
+                    },
+                    "provider": "email",
+                    "created_at": "2024-06-17T00:19:25.774522Z",
+                    "last_sign_in_at": "2024-06-17T00:19:25.774498Z",
```

Comment:
/nit Worried about keeping this in sync with schemachanges but not too big a concern.  I think there's an `is_anonymous` response now

Reply:
Good catch. I will add it in a new PR with further updates. I think in order to keep schema changes in sync we will just have to make sure that the Pydantic model is kept up to date and it's response is always added to the docs.